python/tk_multi_publish2/summary_overlay.py

```diff
@@ -53,6 +53,20 @@ def __init__(self, parent):
         self.ui.info.clicked.connect(self.info_clicked.emit)
         self.ui.publish_again.clicked.connect(self.publish_again_clicked.emit)
 
+    def show_no_items_error(self):
+        """
+        Shows a special message when there is no items and the UI is disabled
+
+        NOTE: an instance of this class attached to
```

Comment:
not sure what this note means. wording?

Reply:
Yes this is incomplete. I was trying to come up with an explanation about the fact that someone needs to create another instance with a different parent BUT it is not the place (this methods) to mention that a different parent is currently required. 
Removing this partial and incomplete comment note, we've already mentioned that in the instance creation in `dialog.py`